routes/user.js

```diff
@@ -149,6 +156,32 @@ router.get('/:id/invitations/in', async (req, res) => {
   }
 });
 
+//GET all user incoming PENDING invitations
+router.get('/:id/invitations/in/pending', async (req, res) => {
+  const userId = req.params.id;
+  try {
+    const user = await User.findById(userId);
+
+    if (!user) {
+      res.status(404).send('User not found');
+    }
+
+    const pendingInvites = await Invitation.find({
+      toEmail: user.email,
+      status: 'pending',
+    });
+
+    if (pendingInvites.length < 1) {
+      return res.status(204).send('No pending invitations found.');
```

Comment:
Maybe we should just send an empty array back so we can just set our invites to that empty array